tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature

```diff
@@ -3778,3 +3778,967 @@ Feature: listing sharedWithMe when auto-sync is disabled
+                      "properties": {
+                        "name": {
+                          "const": "folderToShare"
+                        },
+                        "permissions": {
+                          "minItems": 1,
+                          "maxItems": 1,
+                          "type": "array",
+                          "items": {
+                            "type": "object",
+                            "required": [
+                              "grantedToV2",
+                              "id",
+                              "invitation",
+                              "roles"
+                            ],
+                            "properties": {
+                              "grantedToV2": {
+                                "type": "object",
+                                "required": ["user"],
+                                "properties": {
+                                  "user": {
+                                    "type": "object",
+                                    "required": [
+                                      "displayName",
+                                      "id"
+                                    ],
+                                    "properties": {
+                                      "displayName": {
+                                        "const": "Brian Murphy"
+                                      }
+                                    }
+                                  }
+                                }
+                              },
+                              "invitation": {
+                                "type": "object",
+                                "required": ["invitedBy"],
+                                "properties": {
+                                  "invitedBy": {
+                                    "type": "object",
+                                    "required": ["user"],
+                                    "properties": {
+                                      "user": {
+                                        "type": "object",
+                                        "required": [
+                                          "displayName",
+                                          "id"
+                                        ],
+                                        "properties": {
+                                          "displayName": {
+                                            "const": "Carol King"
+                                          }
+                                        }
+                                      }
+                                    }
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+      """
```

Comment:
It looks like the tests have been duplicated

Reply:
remove that file completely from the PR?

Reply:
seems that the tests have run successfully 😄

Reply:
>seems that the tests have run successfully 😄

because we haven't test in expected failures file from sharedWithMeSyncDisabled.feature

> remove that file completely from the PR?

yeah, I would remove it

Reply:
done.